src/pages/kwonojae/Main/components/Nav.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHome } from '@fortawesome/free-solid-svg-icons';
+import {
+  faPaperPlane,
+  faCompass,
+  faHeart,
+} from '@fortawesome/free-regular-svg-icons';
+import './Nav.scss';
+
+class Nav extends React.Component {
+  render() {
+    return (
+      <nav className="navOJ">
+        <div className="navTitle">
+          <span className="main">westargram</span>
+        </div>
+        <div className="navSearch">
+          <input type="text" placeholder="검색" />
+        </div>
+        <div className="navIcons">
+          <FontAwesomeIcon icon={faHome} className="fas fa-home" />
+          <FontAwesomeIcon icon={faPaperPlane} className="far fa-paper-plane" />
+          <FontAwesomeIcon icon={faCompass} className="far fa-compass" />
+          <FontAwesomeIcon icon={faHeart} className="fas fa-heart" />
```

Comment:
map....?🤔

Reply:
```suggestion
render() {
    const iconList = [
      { id: 1, className: 'fas fa-home' },
      { id: 2, className: 'far fa-paper-plane' },
      { id: 3, className: 'far fa-compass' },
      { id: 4, className: 'fas fa-heart' },
    ];
    
    
      <div className="navIcons">
    {iconList.map(iconInfo => {
      return <i key={iconInfo.id} className={iconInfo.className} />;
          })}
          <img src="/images/kwonojae/profileImage.jpeg" alt="my menu" />
        </div>
```

map 으로 구현해보았습니다. 만, 기존의 fontAwesomeIcon 컴포넌트를 이용한 방식의 경우 icon={} 이 안의 값을 통해 원하는 아이콘을 불러오는 방식인 것 같은데, 중괄호 안에 같은 값을 주거나, 중괄호를 씌워서 줘도 정상적으로 아이콘이 불러와지지 않아서, className 으로 아이콘을 불러올 수 있는 i 태그로 바꿨습니다.

Reply:
아하! 맞아요 그 방법도 있고 패키지로는 저는 아래와 같이 하니까 되더라구요 수정요청은 아니고요!! 그냥 공유드립니다 🙌

```js
// 데이터 객체 안에
icon: faCompass,

// 컴포넌트 map 돌릴때
<FontAwesomeIcon icon={menu.icon} className="블라블라" /> 
```